src/main/java/com/google/cloud/spanner/pgadapter/statements/BackendConnection.java

```diff
@@ -261,15 +261,17 @@ private final class Execute extends BufferedStatement<StatementResult> {
         String command,
         ParsedStatement parsedStatement,
         Statement statement,
-        Function<Statement, Statement> statementBinder) {
-      this(command, parsedStatement, statement, statementBinder, false);
+        Function<Statement, Statement> statementBinder,
+        DatabaseId databaseId) {
+      this(command, parsedStatement, statement, statementBinder, databaseId, false);
     }
 
     Execute(
         String command,
         ParsedStatement parsedStatement,
         Statement statement,
         Function<Statement, Statement> statementBinder,
+        DatabaseId databaseId,
```

Comment:
This change should also either be removed, or moved to a separate PR (but it does not appear to do anything, so removing is probably the best)

Reply:
Removed.